st2api/tests/unit/controllers/v1/test_packs_views.py

```diff
@@ -133,8 +133,8 @@ def test_get_pack_files_and_pack_file_ref_doesnt_equal_pack_name(self):
         # Ref is not equal to the name, controller should still work
         resp = self.app.get("/v1/packs/views/files/dummy_pack_16")
         self.assertEqual(resp.status_int, http_client.OK)
-        self.assertEqual(len(resp.json), 1)
-        self.assertEqual(resp.json[0]["file_path"], "pack.yaml")
+        self.assertEqual(len(resp.json), 3)
+        self.assertIn("pack.yaml", [f["file_path"] for f in resp.json])
```

Comment:
What does this change?

Reply:
The pack in question only had `pack.yaml` before this PR. I added `fixture.py` and `__init__.py`, so the file count goes up to 3.
And, since there is more than one file, we can't just assume that the `resp.json[0]` is the `pack.yaml` file. But is it `[1]` or `[2]`? It's hard to say because the order might depend on the the filesystem. I tried `resp.json[2]` which on GHA was not `pack.yaml`, so they are not sorted alphabetically. To avoid a flaky test relying on a particular sort, I made a list of the files with `[f["file_path"] for f in resp.json]`, and just assert that `pack.yaml` is in it.